core/route/allrouting.js

```diff
@@ -87,6 +87,18 @@ class Routing {
     }
 
     /**
+     * Match Method
+     * @param {String[]} methods - Array of methods to use for a route
+     * @param {String} path - Path for routing
+     * @param {String|Function} controllerAction
+     */
+    match(methods, path, controllerAction) {
+        methods.forEach((method) => {
+            this[method.toLowerCase()](path, controllerAction);
```

Comment:
I would maybe have a constant array in this class something like

```
const allowedMethod = ["get", "post", "patch", "put"];
```

And run `method.toLowerCase()` through that first, just so people know when they typed something wrong.